ARTnews.js

```diff
@@ -0,0 +1,147 @@
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+ 
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+ 
+FW.Scraper({
+itemType         : 'blogPost',
+detect           : FW.Url().match(/\/\d{4}\/\d{2}\//),
+title            : FW.Xpath('//h1[@class="entry-title"]').text(),
+attachments      : [{ url: FW.Url(),
+  title:  "ARTnews snapshot",
+  type: "text/html" }],
+creators         : FW.Xpath('//span[@class="author-meta-name"]/a').text().cleanAuthor("author"),
+date             : FW.Xpath('//span[@class="date-meta"]').text(),
+blogTitle        : "ARTnews",
+language         : "en-US",
+abstractNote     : FW.Xpath('//meta[@name="description"]/@content').text(),
+tags             : FW.Xpath('//span[@class="cat-links"]/a').text()
+});
+
+FW.MultiScraper({
+itemType         : 'multiple',
+detect           : FW.Url().match(/(\/category\/)|(\/\?s=)/), // category or search
+choices          : {
+  titles  :  FW.Xpath('//h2[@class="entry-title"]/a').text(),
+  urls    :  FW.Xpath('//h2[@class="entry-title"]/a').key("href").trim()
+  }
+});
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.artnews.com/category/2017-venice-biennale/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://www.artnews.com/2016/01/23/venice-biennale-taps-christine-macel-to-be-artistic-director-of-2017-edition/",
+		"items": [
+			{
+				"itemType": "blogPost",
+				"title": "Venice Biennale Taps Christine Macel to Be Artistic Director of 2017 Edition",
+				"creators": [
+					{
+						"firstName": "Andrew",
+						"lastName": "Russeth",
+						"creatorType": "author"
+					}
+				],
+				"date": "01/23/16 10:35 am",
+				"abstractNote": "Christine Macel.JEAN-CLAUDE PLANCHET/CENTRE POMPIDOU Today Christine Macel, the chief curator of Paris’s Centre Pompidou, was named artistic director of the",
+				"blogTitle": "ARTnews",
+				"language": "en-US",
+				"url": "http://www.artnews.com/2016/01/23/venice-biennale-taps-christine-macel-to-be-artistic-director-of-2017-edition/",
+				"attachments": [
+					{
+						"title": "ARTnews snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [
+					"2017 Venice Biennale",
+					"News",
+					"The Talent"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.artnews.com/2017/03/10/the-whitney-biennial-arrives-heres-a-round-up-of-coverage-of-artists-in-the-show/",
+		"items": [
+			{
+				"itemType": "blogPost",
+				"title": "The Whitney Biennial Arrives! Here’s a Round-Up of Coverage of Artists in the Show",
+				"creators": [
+					{
+						"firstName": "The Editors of",
```

Comment:
@adam3smith I'm stumped on how to handle single-field authors instead of lastName/firstName and can't find a solution in the documentation. I'm currently shoving the full name into either lastName or firstName. Is there a single-field parameter name I should be using?